Text Component changes has been committed

DeepaliAgrwl opened the pull request.

DeepaliAgrwl committed:
Text Component changes has been committed

rishabh-behl commented on ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml:
```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          jcr:primaryType="nt:unstructured"
+          jcr:title="Text Custom"
```
@DeepaliAgrwl : keep Name as Text only

DeepaliAgrwl replied:
Change done

rishabh-behl commented on ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml:
```diff
@@ -0,0 +1,180 @@
+            </tabs>
+        </items>
+    </content>
+</jcr:root>
```
@DeepaliAgrwl : add source edit functionality too . 
![image](https://user-images.githubusercontent.com/38690824/80563825-c7399580-8a09-11ea-8f05-23f4942dc320.png)

DeepaliAgrwl replied:
@rishabh-behl  added this functionality in both inplace and full screen editing mode.

rishabh-behl commented on ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/text/_cq_dialog/.content.xml:
```diff
@@ -0,0 +1,180 @@
+                                                            </popovers>
+                                                        </inline>
+                                                        <dialogFullScreen
+                                                                jcr:primaryType="nt:unstructured"
```
@DeepaliAgrwl : provide table oftion in fullscreen. 
![image](https://user-images.githubusercontent.com/38690824/80564275-0b796580-8a0b-11ea-8ca1-b45a1b0acf8b.png)

DeepaliAgrwl replied:
Added table option in full screen mode

rishabh-behl reviewed (commented):
@DeepaliAgrwl : pl incorporate comments

DeepaliAgrwl committed:
Review Comments incoporated

1. Added sourceEdit option in plugin editor
2. Changed the name of dialog from Custom Text to Text
3. added table option in fullscreen editing mode

rishabh-behl commented on ui.apps/src/main/content/META-INF/vault/filter.xml:
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
-    <filter root="/apps/hclecomm"/>
+    <filter root="/apps/hclecomm" mode="merge/>
```
@DeepaliAgrwl : build is failing bacuse of this . Please correct with "

rishabh-behl1 committed:
redo chnages in apps filter xml

rishabh-behl commented:
merging
